Functions.Templates/Templates/EventGridBlobTrigger-CSharp-Isolated/.template.config/template.json

```diff
@@ -0,0 +1,77 @@
+{
+    "author": "Microsoft",
+    "classifications": [
+        "Azure Function",
+        "Trigger",
+        "Blob",
+        "EventGrid"
```

Comment:
I would suggest removing event grid if it is not directly using event grid or taking event grid config as part of the template

Reply:
Removed